### PR TITLE
vim,gvim,xxd => 9.x

### PR DESCRIPTION
**Pull request by satmandu**
- Uses exact same logic as `neovim` package PR...

Works properly:
- [x] `x86_64`
- [x] `i686`
- [x] `armv7l` 

### Run the following to get this pull request's changes locally for testing.
```
CREW_TESTING_REPO=https://github.com/satmandu/chromebrew.git CREW_TESTING_BRANCH=vim9 CREW_TESTING=1 crew update
```

### CODE REVIEW

File: packages/gvim.rb [hunk 1]
```diff
@@ -86,11 +98,38 @@ def self.install
...
+        @create_vi_symlink = true
+      else
+        @create_vi_symlink = false
+        puts 'Default vi left unchanged.'.lightgreen
+      end
+    end
+    return unless @create_vi_symlink
+
+    FileUtils.ln_sf "#{CREW_PREFIX}/bin/vim", "#{CREW_PREFIX}/bin/vi"
+    puts 'Default vi set to vim.'.lightgreen
```
Comment (uberhacker) [inline packages/gvim.rb:126]: Why are we symlinking gvim to vi?  Isn't vi a cli command?

File: packages/vim_runtime.rb [hunk 2]
```diff
@@ -128,7 +125,7 @@ def self.install
         silent! call mkdir(expand(&g:undodir), 'p', 0700)
       endif
     EOF
-    system "sed -i 's/set mouse=a/set mouse-=a/g' #{CREW_DEST_PREFIX}/share/vim/vim82/defaults.vim"
+    system "sed -i 's/set mouse=a/set mouse-=a/g' #{CREW_DEST_PREFIX}/share/vim/vim#{@vim_version}/defaults.vim"
```
Comment (uberhacker) [inline packages/vim_runtime.rb:128]: Is `defaults.vim` accessible with this path?

File: packages/gvim.rb (hunk 1 above)
Comment (satmandu) [reply packages/gvim.rb:126]: The gvim package also includes a cli vim binary. (gvim and vim are mutually exclusive.... gvim is basically vim also linked to gtk.)

File: packages/gvim.rb [hunk 3]
```diff
@@ -86,11 +98,38 @@ def self.install
 
   def self.postinstall
     puts
-    puts "The config files are located in #{CREW_PREFIX}/etc".lightblue
-    puts 'User-specific configuration should go in ~/.gvimrc'.lightblue
+    puts "The config files are located in #{CREW_PREFIX}/etc.".lightblue
+    puts 'User-specific configuration should go in ~/.vimrc.'.lightblue
```
[the remark this answers is not in the document]
Comment (satmandu) [reply packages/gvim.rb:102]: gvim specific prefs should go in `.gvimrc` ... technically one should be able to use both....

File: packages/vim_runtime.rb (hunk 2 above)
Comment (satmandu) [reply packages/vim_runtime.rb:128]: Yes.
```
crew files vim_runtime | grep defaults
/usr/local/share/vim/vim90/defaults.vim
```

Comment (satmandu) [reply packages/vim_runtime.rb:128]: (The sed succeeds.)

**Review by uberhacker — APPROVED** [review]

**Pull request merged**